app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -51,7 +51,7 @@ fun ConversationScreen(conversationViewModel: ConversationViewModel) {
         onSendButtonClicked = conversationViewModel::sendMessage,
         onSendAttachment = conversationViewModel::sendAttachmentMessage,
         onDownloadAsset = conversationViewModel::downloadAsset,
-        onImageFullScreenMode = { conversationViewModel.navigateToGallery(it) },
+        onImageFullScreenMode =  conversationViewModel::navigateToGallery ,
```

Comment:
```suggestion
        onImageFullScreenMode = conversationViewModel::navigateToGallery,
```

We have to remove those extra spaces =)